modules/ROOT/pages/install-aws.adoc

```diff
@@ -7,10 +7,10 @@ This topic describes how to install Anypoint Runtime Fabric on your AWS account.
 
 [WARNING]
 ====
-Before installing Runtime Fabric, ensure your infrastructure meets the minimum hardware, operating system, 
-and networking requirements.
+Before installing Runtime Fabric, refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric]. If your infrastructure does not meet the minimum hardware, operating system, 
+and networking requirements, Runtime Fabric will not operate successfully.
 
-Your organization's operations, networking and security teams should review these requirements. See 
+Your organization's operations, networking and security teams must be involved. See 
```

Comment:
```suggestion
Your organization's operations, networking, and security teams must be involved. See 
```
But should we use the exact same names as used earlier: DevOps, Network, and Security?